llama_stack/cli/stack/build.py

```diff
@@ -113,15 +113,11 @@ def _run_stack_build_command_from_build_config(
         if return_code != 0:
             return
 
-        configure_name = (
-            build_config.name
-            if build_config.image_type == "conda"
-            else (f"llamastack-{build_config.name}")
-        )
-        cprint(
-            f"You can now run `llama stack configure {configure_name}`",
-            color="green",
-        )
+        if build_config.image_type == ImageType.conda.value:
+            cprint(
+                f"You can now run `llama stack configure {build_config.name}`",
+                color="green",
+            )
```

Comment:
Why isn't this relevant to `docker`? If you run `llama stack build` for a `docker` image, this next step still seems applicable? I know you don't HAVE to go through this workflow, but some subset still will, right?

The same comment applies in a few other places.

Reply:
or to put it another way -- I think striving for a consistent UX (set of `llama stack ...` commands) for a given workflow would be a nice goal.

Reply:
I've removed it as we want to simplify the dev flow to allow users to directly interact w/ docker images through `docker run` without the need for `llama stack configure + llama stack run`. Added an alternative output message for build w/ docker. 
```
You can now run `docker run -it -p 5000:5000 llamastack-d2` or `llama stack configure llamastack-d2`
```

Reply:
> or to put it another way -- I think striving for a consistent UX (set of `llama stack ...` commands) for a given workflow would be a nice goal.

Thanks for the feedback! The motivation here is to enable the ability s.t. downloading docker image should be sufficient to start Llama Stack server. Instead of the need to run `configure / run` outside of docker container after downloading. We hope this will simplify the user's experience to the standard way of interacting with docker images through `docker/podman`. 

The consistent UX as supported before with `llama stack build -> llama stack configure -> llama stack run` is still supported for docker. Feedbacks are welcome on which way is preferred

(2) / (3) - directly through docker commands. v.s. (4) via `llama stack configure / llama stack run` CLI.

Reply:
Actually I think your suggestion makes more sense, updated to the old output msg!